### DISCUSSION TITLE
BT-1252 Direct Award Justification Previous Procedure Identifier - old procedure?

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#724] [Q&A] [answered]
Hello Labs,

BT-1252: Direct Award Justification Previous Procedure Identifier
The BT-1252's value is a UUID (BT-04) of a previous procedure.
The old, not eforms procedures does not have UUID.

Which field can we use to refer the non eforms procedures which does not have UUID?
Is there any field for this purpose?

Thanks in advance
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi,
you may refer to the TEDXML Result Notice of the previous procedure that justifies the Direct Award by referring to it using the field OPP-090-Procedure.
